src/component/page/deltakerliste-detaljer/filter-meny/FilterMenyStatus.tsx

```diff
@@ -9,6 +9,7 @@ import { FilterMeny } from '../../../felles/table-filter/FilterMeny'
 import { mapTiltakDeltakerStatusTilTekst } from '../../../../utils/text-mappers'
 import { FiltermenyDataEntry } from '../../../felles/table-filter/filtermeny-data-entry'
 import { klikkFilterMeny, loggKlikk } from '../../../../utils/amplitude-utils'
+import useLocalStorage from '../../../../hooks/useLocalStorage'
```

Comment:
Her er ikke sonarcloud fornøyd, for det ble duplisert kode  for denne komponenten mot den andre listas status-filter-komponent